cmd/nitrite/main_test.go

```diff
@@ -1,56 +1,39 @@
 package main_test
 
 import (
+	"bytes"
 	"encoding/base64"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"os"
+	"os/exec"
 	"testing"
 	"time"
 
-	"github.com/ionrock/procs"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/blocky/nitrite"
 	"github.com/blocky/nitrite/internal"
 )
 
-func runNitriteWithEnv(
-	cmd string,
-	env map[string]string,
-) (nitrite.Document, error) {
-	nitriteProc := procs.NewProcess(cmd)
-	nitriteProc.OutputHandler = func(line string) string { return line }
-	nitriteProc.ErrHandler = func(line string) string { return line }
-	nitriteProc.Env = procs.ParseEnv(os.Environ())
-	for k, v := range env {
-		nitriteProc.Env[k] = v
-	}
-
-	err := nitriteProc.Start()
-	if err != nil {
-		return nitrite.Document{},
-			fmt.Errorf("starting nitrite process: %w", err)
-	}
-
-	err = nitriteProc.Wait()
-	if err != nil {
-		errOut, errOutErr := nitriteProc.ErrOutput()
-		err = errors.Join(err, errOutErr)
-		return nitrite.Document{},
-			fmt.Errorf("waiting on nitrite process '%s': %w", errOut, err)
-	}
-
-	outBytes, err := nitriteProc.Output()
-	if err != nil {
-		return nitrite.Document{},
-			fmt.Errorf("getting nitrite process output: %w", err)
+func runNitrite(cmd string) (nitrite.Document, error) {
```

Comment:
Updated interface to remove env.  All callers just pass nil.

Reply:
Great